examples/README.md

```diff
@@ -0,0 +1,279 @@
+    json_object_set_string(root_object, "name", asset_name.c_str());
+    json_object_set_number(root_object, "value", value);
+    char* serialized_string = json_serialize_to_string(root_value);
+    std::string out(serialized_string);
+    json_free_serialized_string(serialized_string);
+    json_value_free(root_value);
+    return out;
+}
+
+```
+### Build 
+To build the chaincode, add `CMakefile.txt` and `Makefile` (Note that CMakeLists.txt is a modified version of examples/auction/CMakeLists.txt.  Makefile is the same as in examples/auction/Makefile).
+File: CMakeLists.txt
+```
+cmake_minimum_required(VERSION 3.5.1)
+
+set(SOURCE_FILES
+    helloworld_cc.cpp
+    )
+
+include(../../ecc_enclave/enclave/CMakeLists-common-app-enclave.txt)
+```
+
+
+Make sure you have the environment variables set.  Now, execute:
+
+
+```make```
+
+
+If you dont see any errors, you are good to go.
+
+Optional: For this `helloworld` chaincode to be included in the build framework, modify the file: `FPC-INSTALL-DIR/examples/Makefile` to include the `helloworld` chaincode.
+
+File: Makefile   
+```TOP = ..
+include $(TOP)/build.mk
+
+EXAMPLES = auction echo helloworld
+
+build clean:
+	$(foreach DIR, $(EXAMPLES), $(MAKE) -C $(DIR) $@ || exit;)
+```
+  
+Next step is to install, instantiate the chaincode and invoke transactions.  For this, 
+you use the integration framework.  Move to `FPC-INSTALL-DIR/integration`.   In the existing `Makefile`,  make these modifications: 
+
+```...
+test: auction_test echo_test deployment_test helloworld_test
+...
+helloworld_test:
+        ./helloworld_test.sh
+```        
+### Time to test !  
+Create a file to install, instantiate the chaincode, submit transactions.  The following test script invokes the transaction, putData thrice;  after each time, invokes getData to read the value set before. 
+In `FPC-INSTALL-DIR/integration` folder, Create file `helloworld_test.sh`
+```
+
+SCRIPTDIR="$(dirname $(readlink --canonicalize ${BASH_SOURCE}))"
+FPC_TOP_DIR="${SCRIPTDIR}/.."
+CONFIG_HOME="${SCRIPTDIR}/config"
+FABRIC_SCRIPTDIR="${FPC_TOP_DIR}/fabric/bin/"
+
+. ${FABRIC_SCRIPTDIR}/lib/common_utils.sh
+. ${FABRIC_SCRIPTDIR}/lib/common_ledger.sh
+
+CC_ID=helloworld_test
+
+#this is the path that will be used for the docker build of the chaincode enclave
+ENCLAVE_SO_PATH=examples/helloworld/_build/lib/
+
+CC_VERS=0
+num_rounds=3
+
+helloworld_test() {
+    # install, init, and register (auction) chaincode
+    try ${PEER_CMD} chaincode install -l fpc-c -n ${CC_ID} -v ${CC_VERS} -p ${ENCLAVE_SO_PATH}
+    sleep 3
+
+    try ${PEER_CMD} chaincode instantiate -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -v ${CC_VERS} -c '{"args":["init"]}' -V ecc-vscc
+    sleep 3
+
+    try ${PEER_CMD} chaincode invoke -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["setup", "ercc"]}' --waitForEvent
+
+    try ${PEER_CMD} chaincode query -o ${ORDERER_ADDR} -C ${CHAN_ID} -n ${CC_ID} -c '{"Args":["getEnclavePk"]}'
```

Comment:
getEnclavePK i would just drop. It's rather unmotivated here and is completely unused

Reply:
Will remove this query to getEnclavePK.